lib/eventboss/queue_listener.rb

```diff
@@ -1,9 +1,33 @@
 module Eventboss
   class QueueListener
     class << self
+      def select(
+        include: Eventboss.configuration.listeners[:include],
```

Comment:
I'm not a huge fan of defaults here, since it creates coupling to configuration. Would be fine w/ me if laucher passed those args explicitly.